lldb/source/Plugins/Process/Windows/Common/TargetThreadWindows.cpp

```diff
@@ -33,6 +34,9 @@
 using namespace lldb;
 using namespace lldb_private;
 
+using GetThreadDescriptionFunctionPtr = HRESULT
+WINAPI (*)(HANDLE hThread, PWSTR *ppszThreadDescription);
+
```

Comment:
Can we just `#include <processthreadsapi.h>` and avoid manually declaring a function prototype?